va/va.go

```diff
@@ -22,6 +24,7 @@ import (
 	"github.com/letsencrypt/pebble/acme"
 	"github.com/letsencrypt/pebble/ca"
 	"github.com/letsencrypt/pebble/core"
+	"github.com/miekg/dns"
```

Comment:
Is there a reason to use miekg/dns here instead of https://golang.org/pkg/net/#LookupTXT ?